src/Tools/Extensions.ApiDescription.Client/src/build/Microsoft.Extensions.ApiDescription.Client.props

```diff
@@ -98,6 +93,13 @@
 
     <!-- OpenApiProjectReference items may include all OpenApiReference metadata. -->
     <OpenApiProjectReference>
+      <!--
+        Semicolon-separated list of global properties to remove in a @(ProjectReference) item created for this
+        @(OpenApiProjectReference). These properties, along with Configuration, Platform, RuntimeIdentifier and
+        TargetFrameworks, are also removed when invoking the 'OpenApiGetDocuments' target in the referenced project.
+      -->
+      <GlobalPropertiesToRemove>TargetFramework</GlobalPropertiesToRemove>
```

Comment:
@rainersigwald is this enough in both contexts where the metadata is used (especially where additional properties aren't always listed)?

Reply:
I'm not sure I understand the question, can you elaborate?

The implementation in this PR looks ok to me. Another option would be using the metadata `UndefineProperties`, which is handled internally by the `MSBuild` task and so would come along for free in any invocation. But this is The Standard Pattern.

The only downside I can think of to setting this in the item definition is that it becomes easier to accidentally override. But it also becomes possible to _intentionally_ override, which seems worth it.

Reply:
Your answer covered my questions 😃